packages/commonwealth/server/controllers/server_threads_methods/get_bulk_threads.ts

```diff
@@ -238,7 +241,9 @@ export async function __getBulkThreads(
 
   const countsQuery = `
      SELECT id, title, stage FROM "Threads"
-     WHERE chain = $chain AND (stage = 'proposal_in_review' OR stage = 'voting')`;
+     WHERE ${
+       chain ? 'chain = $chain AND' : ''
+     } (stage = 'proposal_in_review' OR stage = 'voting')`;
```

Comment:
Not your problem for right now but this needs to change, as stages should be editable by admins so we have no guarantee that "proposal_in_review" or "voting" will be valid stages.